projects/observability/src/shared/components/utils/d3/d3-visualization.scss

```diff
@@ -17,7 +17,7 @@
     flex-direction: row-reverse;
     .chart-legend-container {
       margin-left: 12px;
-      margin-right: 12px;
+      margin-right: 36px;
```

Comment:
this may not be the case for all legend orientations. We should only apply larger right migin when we have left aligned legend

Reply:
Yeah this only the case for left aligned, `row-reverse` class will only be applied when it is left aligned

Reply:
in `d3-visualization.builder`

```ts
protected getOuterContainerLayoutClass(configuration: TInternalConfig): string {
    switch (configuration.legend) {
      case LegendPosition.Bottom:
        return 'column';
      case LegendPosition.Top:
      case LegendPosition.TopLeft:
      case LegendPosition.TopRight:
        return 'column-reverse';
      case LegendPosition.Right:
        return 'row';
      case LegendPosition.Left:
        return 'row-reverse';
      case LegendPosition.None:
        return '';
      default:
        return assertUnreachable(configuration.legend);
    }
  }
```